### PR TITLE
Stagin fixes

### PR DESCRIPTION
**Pull request by Bod9001**
### Changelog:
CL: [Fix] Fixed mouse drag being invisible
CL: [Balance] DNA dinosaur egg puzzle made simpler 
CL: [Balance]  You can now rotate while resisting cuffs.
CL: [Improvement] Medical dispenser now dispenses Prescription glasses
CL: [Fix] Buffs and fixes Reagentmetabolism is now 4x effective and ignores damage
CL: [Fix] Fixed pill bottles and other containers being able to store everything.
CL: [Fix] Fixed rubbish bags being able to be opened

### CODE REVIEW

File: UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/ExternalBodyHealthEffect.cs [hunk 1]
```diff
@@ -52,15 +52,15 @@ public class ExternalBodyHealthEffect : MetabolismReaction
 						foreach (var Effect in Effects)
 						{
 							bodyPart.RelatedPart.TakeDamage(null,
-								Effect.EffectPerOne * ReagentMetabolismMultiplier * -OverdoseDamageMultiplier,
+								Effect.EffectPerOne * -OverdoseDamageMultiplier * ReagentMetabolismMultiplier,
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/ExternalBodyHealthEffect.cs:55]: isn't this exactly the same?

File: UnityProject/Assets/Scripts/Systems/Inventory/Structure/DefinedSlotCapacity.cs [hunk 2]
```diff
@@ -42,6 +42,8 @@ public class DefinedSlotCapacity : SlotCapacity
 	public override bool CanFit(Pickupable toCheck)
 	{
 		if (toCheck == null) return false;
+		bool ToReturn = true;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Systems/Inventory/Structure/DefinedSlotCapacity.cs]:
```suggestion
		bool canFit = true;
```
``ToReturn`` says nothing about what the bool is supposed to mean

File: UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/BodyHealthEffect.cs [hunk 3]
```diff
@@ -96,17 +96,17 @@ public struct TypeAndStrength
...
 			}
 
-			BodyReactionAmount = ProcessingAmount * ReagentMetabolismMultiplier;
+			BodyReactionAmount = ProcessingAmount;
 		}
 
 		foreach (var bodyPart in Toloop)
 		{
-			var Individual = bodyPart.ReagentMetabolism * bodyPart.BloodThroughput * bodyPart.CurrentBloodSaturation * Mathf.Max(0.10f, bodyPart.RelatedPart.TotalModified) * ReagentMetabolismMultiplier;
+			var Individual = bodyPart.ReagentMetabolism * bodyPart.BloodThroughput * bodyPart.CurrentBloodSaturation;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/BodyHealthEffect.cs:104]: why remvoe the second multiplication?

**Comment by corp-0** [comment]
> CL: [Fix] Buffs and fixes Reagentmetabolism

this change doesn't say much

File: UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/BodyHealthEffect.cs (hunk 3 above)
Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/BodyHealthEffect.cs:104]: what gilles asked

File: UnityProject/Assets/Scripts/Systems/Inventory/Structure/DefinedSlotCapacity.cs (hunk 2 above)
Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Systems/Inventory/Structure/DefinedSlotCapacity.cs]: `ToReturn` can be renamed to `HasRequiredTraits`

File: UnityProject/Assets/Scripts/HealthV2/Living/PolymorphicSystems/ChemicalMetabolismSystem.cs [hunk 4]
```diff
@@ -145,13 +145,12 @@ public void MetaboliseReactions()
...
 				{
 					ProcessingAmount += metabolismComponent.ReagentMetabolism * metabolismComponent.BloodThroughput *
-					                    metabolismComponent.CurrentBloodSaturation *
-					                    Mathf.Max(0.10f, metabolismComponent.RelatedPart.TotalModified);
+					                    metabolismComponent.CurrentBloodSaturation;
 				}
 
 				if (ProcessingAmount == 0) continue;
 
-				//Reaction.React(PrecalculatedMetabolismReactions[Reaction], _reagentPoolSystem.BloodPool, ProcessingAmount);
```
Comment (MaxIsJoe) [inline UnityProject/Assets/Scripts/HealthV2/Living/PolymorphicSystems/ChemicalMetabolismSystem.cs:154]: https://user-images.githubusercontent.com/34368774/230813874-0c442cca-92d5-446c-b142-05059c97f1d8.mp4

**Review by MaxIsJoe — CHANGES_REQUESTED** [review]

File: UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/ExternalBodyHealthEffect.cs (hunk 1 above)
Comment (Bod9001) [reply UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/ExternalBodyHealthEffect.cs:55]: yeah, was fiddling around with the calculations, came back to the same thing

File: UnityProject/Assets/Scripts/HealthV2/Living/PolymorphicSystems/ChemicalMetabolismSystem.cs (hunk 4 above)
Comment (Bod9001) [reply UnityProject/Assets/Scripts/HealthV2/Living/PolymorphicSystems/ChemicalMetabolismSystem.cs:154]: yeah, it was commented out when the system was implemented but not turned on xD

File: UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/BodyHealthEffect.cs (hunk 3 above)
Comment (Bod9001) [reply UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/BodyHealthEffect.cs:104]: to make it so body part  Damage doesn't slow down medical chemistry, E.g scenarios where you're trying to heal heavily damaged limb but it's damage so the chemistry doesn't work so well

**Comment by Bod9001** [comment]
good to rereview

**Pull request merged**